tests/pytorch_pfn_extras_tests/onnx_tests/test_export_testcase.py

```diff
@@ -254,7 +256,8 @@ def test_export_testcase_strip_large_tensor_data():
 
     output_dir = _helper(
         model, x, 'mnist_stripped_tensor_data',
-        output_grad=True, strip_large_tensor_data=True)
+        output_grad=True, strip_large_tensor_data=True,
+        metadata=True)
```

Comment:
```suggestion
        metadata=False)
```
Sorry, did you mean this? 😅

Reply:
This test is the only test that needed metadata so I'm enabling it explicitly
(Striped test case will have metadata to tell it. Though this needs to be warned when `metadata=False` and `strip_large_tensor_data=True` are both enabled)

Reply:
ok! I understood the contrary hahaha. Thanks!